.github/workflows/ci.yml

```diff
@@ -97,3 +97,54 @@ jobs:
 
       - name: Run unit tests
         run: npm test
+
+  vrt-runner:
```

Comment:
Can we add a name here that is more descriptive? So that it does not say `CI / vrt`